libs/vscode-ui/components/src/lib/multiple-select/multiple-select.component.ts

```diff
@@ -19,21 +17,23 @@ import { Subscription } from 'rxjs';
   styleUrls: ['./multiple-select.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class MultipleSelectComponent implements OnInit, OnChanges, OnDestroy {
+export class MultipleSelectComponent implements OnInit, OnDestroy {
   @Input() field: Option;
   @Input() value: string[];
   @Output() readonly valueChange = new EventEmitter<string[]>();
-  selectControl = new FormControl([]);
+  selectControl: FormControl;
   items: string[];
   parentFormGroup: FormGroup;
 
-  constructor(private readonly controlContainer: ControlContainer) {
-    this.parentFormGroup = this.controlContainer.control as FormGroup;
-  }
+  constructor(private readonly controlContainer: ControlContainer) {}
 
   private readonly subscriptioins = new Subscription();
```

Comment:
no problem. Honestly I debated just deleting this code because nothing actually uses the output this Subscription is set up for. All the components have change Output but none are using it because they were also all hooked into Reactive forms. I can see why you might want generic components that can do both Template and Reactive forms, but it's just never gonna be used here. Anyway, I'll fix the spelling for sure!